Updates the Install Minder Server documentation (#3880)
stacklok/minder: docs/docs/run_minder_server/intro_to_install.md

---
title: Introduction to running Minder
sidebar_position: 5
---

# Introduction to running Minder

> Minder is platform, comprising of a control plane, a CLI, a database and an identity provider.

Reviewer: We should mention OpenFGA (permissions service) as well.

The control plane runs two endpoints, a gRPC endpoint and a HTTP endpoint.

> Minder is controlled and managed via the CLI application `minder`.

Reviewer: ```suggestion
Users interact with the Minder server via the CLI application `minder`.
```

The previous sentence suggested that `minder` was used to start and stop the server.

PostgreSQL is used as the database.

> Keycloak is used as the identity provider.

Reviewer: Again, mention OpenFGA as a permissions service.
